tests/conftest.py

```diff
@@ -226,6 +225,16 @@ def tf_version(request):
     return request.param
 
 
+@pytest.fixture(scope="module", params=["py2", "py3"])
+def tf_py_version(tf_version, request):
+    version = [int(val) for val in tf_version.split(".")]
+    if version < [1, 13]:
+        return "py2"
```

Comment:
1.11-1.12 have Python 3 support (those are the two versions with both legacy and current TF support)

Reply:
scriptmode, thank you.